DigitalLearningSolutions.Web/Views/ApplicationSelector/Index.cshtml

```diff
@@ -0,0 +1,91 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.ApplicationSelector
+@using Microsoft.Extensions.Configuration
+
+@inject IConfiguration Configuration
+
+@model ApplicationSelectorViewModel
+
+@{
+  ViewData["Title"] = "Switch Application";
+}
+
+<div class="nhsuk-grid-row">
+  @if (Model.LearningPortalAccess)

```

Comment:
Ah, could we also add a "Switch application" h1 (I think it's required for accessibility scans to pass)?